backend-api/template.yaml

```diff
@@ -1441,6 +1455,250 @@ Resources:
           ReturnData: false
           Expression: (errorSum5XX/invocations) * 100
 
+  AsyncToken4XXLowThresholdAlarm:
+    Condition: DeployAlarms
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmDescription: "1% or more of requests are failing on the /async/token endpoint with a 4XX error. See runbook: https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/4800446694/Alarms+Runbook+Warnings"
+      AlarmName: !Sub "${AWS::StackName}-LowThresholdAsyncToken4XXApiGwAlarm"
```

Comment:
Question: do you feel it's useful to have the 'Alarm' suffix in the alarm name? I know it's an existing convention, but whenever it fires we know it's an alarm. I'm on the fence.

Reply:
I don't think it does any harm, but I'd lean towards removing it anyway since as you say it doesn't add any information.

Reply:
While we're at it in terms of renaming - most of our AWS resource names (lambdas, buckets, api gateways) use kebab-case throughout, and of course the stack name prefix will be in kebab-case. Should we change this to `${AWS::StackName}-low-threshold-async-token-4xx-api-gw` for consistency?

Reply:
I would prefer kebab case in the template. I think it helps with readability; it's a good suggestion.

Reply:
Have changed the names as above